NineChronicles.Standalone.Executable/Program.cs

```diff
@@ -83,6 +83,8 @@ static void ConfigureSentryOptions(SentryOptions o)
                     "The number of required confirmations to recognize a block.  0 by default."
             )]
             int confirmations = 0,
+            [Option("validate", Description = "Flag to turn on validating action renderer.")]
```

Comment:
How about `validateRender` instead of `validate`?

Reply:
How about `--strict-rendering`?